src/build/chrome.manifest.json

```diff
@@ -1,5 +1,5 @@
 {
-	"version": "1.2.45",
+	"version": "1.2.46_Chrome",
```

Comment:
looks like Chrome doesn't accept this format. pls use just numbers.

Reply:
Right, they have a separate field for "version_name". https://developer.chrome.com/extensions/manifest/version

I'll just stick to the existing format for this.